Development-Guidelines/building-angular-project.md

```diff
@@ -14,10 +14,10 @@ The bottom line is the UI project has zero dependencies on asp.net or Windows. H
 Umbraco 7 needs a couple of things to run:
 
 ### Node.js 
-To compile and run the UI project you need node.js installed, you can get that at [http://nodejs.org](nodejs.org) for both Windows and OSX.
+To compile and run the UI project you need node.js installed, you can get that at [nodejs.org](http://nodejs.org) for both Windows and OSX.
 
 ### Grunt
-When you have node.js installed, you need to install Grunt. Grunt is a simple JavaScript task runner, basically like Nant, Msbuild or any traditional build system [http://gruntjs.com](more about grunt here).
+When you have node.js installed, you need to install Grunt. Grunt is a simple JavaScript task runner, basically like Nant, Msbuild or any traditional build system. You can [read more about grunt here](http://gruntjs.com).
```

Comment:
Can we find something else that "here" as a link text?  Eg same as nodeJs fix above?

Reply:
Could be something like shown below?

![image](https://user-images.githubusercontent.com/3634580/40588136-2c6f0c6a-61d9-11e8-8ccb-c8235dfb86df.png)

Reply:
Would be better.  Mentioning that you are going to another website in this case is much better!

Reply:
Should be fixed now ;)